modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/ad/AdColors.kt

```diff
@@ -0,0 +1,58 @@
+package au.com.shiftyjelly.pocketcasts.compose.ad
+
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.graphics.Color
+import au.com.shiftyjelly.pocketcasts.compose.theme
+
+data class AdColors(
+    val bannerAd: Banner,
+) {
```

Comment:
This type looks a little bit wacky now because in one of the future PRs I'll introduce `ReportSheet` colors that'll be a separate field in `AdColors`.